galley/pkg/runtime/publish/strategy.go

```diff
@@ -31,18 +33,18 @@ const (
 	defaultTimerFrequency = 500 * time.Millisecond
 )
 
-// publishingStrategy is a heuristic model for deciding when to publish snapshots. It tries to detect
+// PublishingStrategy is a heuristic model for deciding when to publish snapshots. It tries to detect
 // quiesce points for events with a total bounded wait time.
-type publishingStrategy struct {
+type PublishingStrategy struct {
```

Comment:
Can we reduce the refactorings in 1.1? I am not against the spirit of the change, (I'd in fact say, "also rename this to "Strategy" normally). But I want to minimize the total changes going in 1.1 as much as possible at this point. We can do such refactorings in 1.2